components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -0,0 +1,122 @@
++++
+title = "Client Runs"
+
+date = 2021-03-12T12:01:58-07:00
+draft = false
+
+gh_repo = "automate"
+[menu]
+  [menu.automate]
+    title = "Chef Infra Server"
+    parent = "automate"
+    identifier = "automate/chef_infra_server.md Chef Infra Server"
+    weight = 60
++++
+
+## Overview
+
+The __Chef Server__ page shows all the Chef Infra Servers connected to Chef Automate. The __Chef Server__ user interface can be used to view and manage all the Chef Infra Server Objects like:
+
+- Cookbooks
+- Roles
+- Environments
+- Data Bags
+- Clients
+
+## Connect to the Chef Infra Server
+
+To start making use out of the Chef Infra Server, firstly deploy and run the [Chef Automate](https://docs.chef.io/automate/install/#installation-guide).
+
+**Note:** No explicit configuration is required to install the Automate Infra views.
+
+Login to Chef Automate using the credentials. Once logged in, the page looks like as shown below:
+
+![Chef Automate](/images/automate/chef-automate-on-chef-infra-page.png)
+
+Click on **Infrastructure** located at the top bar of the page. Refer to the image below:
+
+![Chef Automate with Highlighted Infrastructure](/images/automate/chef-automate-infrastructure.png)
+
+Click on the **Chef Servers** located at the left panel of the page. The page looks like as shown below:
+
+![Chef Infra Server](/images/automate/chef-server-page.png)
+
+The above image is the **Chef Server** page which contains all the Chef Infra Servers connected to the Chef Automate. The Chef Infra Servers listed are the ones that are already configured and added to the Chef Automate. To create a new Infra Server click [here](https://docs.chef.io/automate/infra_server/).
+
+To add the created Chef Server to the Infrastructure, click on **Add Chef Server** button. Refer to the image below:
+
+![Add Chef Server](/images/automate/add-chef-server-button.png)
+
+Clicking on the **Add Chef Server** button opens a popup menu as shown below:
```

Comment:
Select **Add Chef Server** and ...
Using "select" applies to all instances of "click on". 

Explain the fields that the user needs to complete. Emphasize the actions that a user needs to take instead of the UI. If you feel we need to mention the popup, use "Selecting **Add Chef Sever** opens a form" or "helper window", which we tried using in the past (I'm not a fan of that one, but it works).

Use example company from the docs style guide (4th Cafe, 4thcafe.com, ip: 192.0.2.0) (The ip for example domains in [rfc5737](https://tools.ietf.org/html/rfc5737)).